utilities/post_migration.py

```
@@ -1230,6 +1231,72 @@ def check_ssl_configuration(source_provider: BaseProvider) -> None:
+    """
+    vm_name = vm["name"]
+    destination_vm_name = resolve_destination_vm_name(vm)
+
+    # Get source VM data from plan
+    source_vm_data = plan.get("source_vms_data", {}).get(vm_name, {})
+
+    # Check if Windows VM
+    if source_vm_data.get("win_os", False):
+        raise ValueError(
+            f"Command output verification is only supported on Linux guests. VM '{vm_name}' is a Windows guest."
+        )
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)